frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -176,6 +179,9 @@ const enableImplicitActionsForModel =
     dispatch(Actions.actions.invalidateLists());
   };
 
+const CREATE_PUBLIC_LINK = "metabase/action/CREATE_PUBLIC_LINK";
+const DELETE_PUBLIC_LINK = "metabase/action/DELETE_PUBLIC_LINK";
```

Comment:
If we're implementing this as an entity action, I think we should follow the naming pattern here:

```
metabase/entities/actions/CREATE_PUBLIC_LINK
metabase/entities/actions/DELETE_PUBLIC_LINK
```

?

Reply:
Oh yeah, totally forgot about that. In the first version, I created these actions outside the entity like for public dashboards and questions.

https://github.com/metabase/metabase/blob/2ee1479da12244f84f402b1c0d93bd1df811f191/frontend/src/metabase/dashboard/actions/sharing.js#L35